doc/man/scons.xml

```diff
@@ -1120,24 +1120,48 @@ the mechanisms in the specified order.</para>
   </listitem>
   </varlistentry>
 
+  <varlistentry id="opt--experimental">
+    <term>
+      <option>--experimental=<replaceable>all</replaceable></option>,
+      <option>--experimental=<replaceable>none</replaceable></option>
+    </term>
+    <listitem>
+      <para>Enable experimental features and/or tools. <emphasis role="strong">No Support offered for any features or tools enabled by this flag</emphasis></para>
+      <para>The default setting is none.</para>
```

Comment:
Okay so the question I had the other day (are experimental features comma-separated strings, or are they additive over several calls to `--experimental`) is "neither?  It's either on, or off.  In that case I'd use the values `enabled` and `disabled` or the simpler `on` and `off` rather than `all` and `none`.

Reply:
any of the following should work.

```
--experimental=all
--experimental=none
--experimental=feata,featb (should be both enabled0
--experimental=feata --experimental=featb (should be both)
```